### ISSUE TITLE
Fatal error

### ISSUE THREAD

**Issue by lennynld** [janizde/WP-Opening-Hours#8]
can't activate version 2.0 because of following error's

Warning: require_once(includes/admin-notice-helper/admin-notice-helper.php): failed to open stream: No such file or directory in /public_html/wordpress/wp-content/plugins/WP-Opening-Hours-master/bootstrap.php on line 24

Fatal error: require_once(): Failed opening required 'includes/admin-notice-helper/admin-notice-helper.php' (include_path='.:/usr/share/php') in /public_html/wordpress/wp-content/plugins/WP-Opening-Hours-master/bootstrap.php on line 24

**Comment by janizde** [comment]
Please mind, that when cloning or forking the GitHub repo you will also have to add the submodules with 
`git submodule init`
You will also have to install the npm modules and compile the css and js with grunt.

If you just want to test, you may also use the .zip at http://www.jannikportz.de/2015/04/20/opening-hours-2-0-beta-please-test/ it already includes the submodules and compiled js and css

**Comment by lennynld** [comment]
after activation the plugin
Fatal error: Uncaught exception 'Exception' with message 'DateTimeZone::__construct(): Unknown or bad timezone ()' in /home/loveyoudesign.com/public_html/wordpress/wp-content/plugins/wp-opening-hours/classes/OpeningHours/Module/I18n.php:110 Stack trace: #0 /home/loveyoudesign.com/public_html/wordpress/wp-content/plugins/wp-opening-hours/classes/OpeningHours/Module/I18n.php(110): DateTimeZone->__construct('') #1 [internal function]: OpeningHours\Module\I18n::init('') #2 /home/loveyoudesign.com/public_html/wordpress/wp-includes/plugin.php(496): call_user_func_array(Array, Array) #3 /home/loveyoudesign.com/public_html/wordpress/wp-settings.php(353): do_action('init') #4 /home/loveyoudesign.com/public_html/wordpress/wp-config.php(80): require_once('/home/loveyoude...') #5 /home/loveyoudesign.com/public_html/wordpress/wp-load.php(37): require_once('/home/loveyoude...') #6 /home/loveyoudesign.com/public_html/wordpress/wp-admin/admin.php(31): require_once('/home/loveyoude...') #7 /home/loveyoudesign.com/public_html/wordpress/wp- in /home/loveyoudesign.com/public_html/wordpress/wp-content/plugins/wp-opening-hours/classes/OpeningHours/Module/I18n.php on line 110

**Comment by fotomut** [comment]
I get the same fatal error

```
[08-Jul-2015 18:16:35 UTC] PHP Fatal error:  Uncaught exception 'Exception' with message 'DateTimeZone::__construct(): Unknown or bad timezone ()' in /var/www/html/wp-content/plugins/wp-opening-hours/classes/OpeningHours/Module/I18n.php:110
Stack trace:
#0 /var/www/html/wp-content/plugins/wp-opening-hours/classes/OpeningHours/Module/I18n.php(110): DateTimeZone->__construct('')
#1 [internal function]: OpeningHours\Module\I18n::init('')
#2 /var/www/html/wp-includes/plugin.php(496): call_user_func_array(Array, Array)
#3 /var/www/html/wp-settings.php(354): do_action('init')
#4 /var/www/html/wp-config.php(96): require_once('/var/www/html/w...')
#5 /var/www/html/wp-load.php(37): require_once('/var/www/html/w...')
#6 /var/www/html/wp-admin/admin.php(31): require_once('/var/www/html/w...')
#7 /var/www/html/wp-admin/plugins.php(10): require_once('/var/www/html/w...')
#8 {main}
  thrown in /var/www/html/wp-content/plugins/wp-opening-hours/classes/OpeningHours/Module/I18n.php on line 110
```

**Comment by fotomut** [comment]
I fixed the error on my system by explicitly setting the timezone_string option using the following sql statement `update wp_options set option_value = 'America/New_York' where option_name = 'timezone_string';`

However, I think the plugin should be able to handle when a site has no value set for `timezone_string` option, as was the case for me.  Perhaps default to `UTC`?  See here for supported values http://php.net/manual/en/timezones.php

**Comment by janizde** [comment]
solved in 685ba56e152be8bf0a90c2cbacab42df8572334c
it now takes the php default value if neither gmt_offset nor timezone_string are set
